src/contrib/cluster/Akka.DistributedData.Tests.MultiNode/Akka.DistributedData.Tests.MultiNode.csproj

```diff
@@ -4,6 +4,7 @@
   <PropertyGroup>
     <AssemblyTitle>Akka.DistributedData.Tests.MultiNode</AssemblyTitle>
     <TargetFrameworks>$(NetFrameworkTestVersion);$(NetTestVersion);$(NetCoreTestVersion)</TargetFrameworks>
+    <Platforms>x64</Platforms> <!-- target x64 only due to LightningDb support -->
```

Comment:
I think `AnyCPU` might have something to do with the issues here. Going to see if I can get this all to work under x64.

Reply:
Doesn't appear to have helped, based on the data from my local machine

Reply:
LOL, applied this to the wrong project....